application/listeners/posthelpers_fixtures_test.go

```diff
@@ -62,6 +62,13 @@ func CreateFixtures_GetPostRecipients(ms *ModelSuite, t *testing.T) orgUserPostF
 		}
 	}
 
+	_, err := users[1].UpdateStandardPreferences(models.StandardPreferences{Language: domain.UserPreferenceLanguageFrench})
+
+	if err != nil {
+		t.Errorf("could not create language preference for user %v ... %v.", users[1].Nickname, err)
```

Comment:
```suggestion
		ms.NoErr(err, "could not create language preference for user %v ... %v.", users[1].Nickname, err)
```
and remove the `if` and `FailNow()` here and elsewhere in this function.

Reply:
Good catch